src/core/createEvent.js

```diff
@@ -52,6 +52,13 @@ export default () => {
 
       return userXdm.web.webPageDetails.viewName;
     },
+    getConsent() {
+      if (!userXdm || !userXdm.consentStrings) {
```

Comment:
Russ is still working on the XDM mixin for consent and preferences. I'm not sure where it will be, but this is the location for IAB consent strings.